Cargo.lock

```diff
@@ -283,9 +285,9 @@ dependencies = [
 
 [[package]]
 name = "predicates-core"
-version = "1.0.0"
+version = "1.0.3"
```

Comment:
Hm looks like cargo did some internal magic even though the `cargo.toml` was not updated.
Ideally this should be in a separate PR, so investigate what went wrong here.